internal/cmd/local/command_kafka.go

```diff
@@ -0,0 +1,30 @@
+package local
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+)
+
+const imageName = "523370736235.dkr.ecr.us-west-2.amazonaws.com/confluentinc/kafka-local:latest"
+const urlPrefix = "http://0.0.0.0:%s"
+
+type kafkaCommand struct {
+	*pcmd.CLICommand
+}
```

Comment:
None of these commands should require authentication, so we only need one command `struct` that can be defined in internal/cmd/local/command.go